### ISSUE TITLE
Daily meeting (15/12/2016)

### ISSUE THREAD

**Issue by Lai0723** [Lai0723/Gallery-Shopping-Website#4]
So how was your progress on the module guys?

**Comment by Lai0723** [comment]
For me i just finished the table for storing role of the user and today i will working on the changing password function for the user to change the account password after he or she login.

**Comment by kcchan3012** [comment]
For me i just done with the menu page and other page and I link it with the database and added to the master page , you guys can go have a look. If got any problem please tell me as soon as possible.

**Comment by pqyyieeuyi** [comment]
Boon Seng do you done with the registration part? Can send it to me?

**Comment by boonsenglim** [comment]
No, i just facing some problem which i dont know how to store the data to the databse. Anyone have any idea?

**Comment by Lai0723** [comment]
Oh about that thing you need to write query code when the user key in all the field then you store the username into the database like this ( field name = the textbox name ) and do the same for the password.

**Comment by pqyyieeuyi** [comment]
So if like the field is user name and the text box name is T.text then we should write it like this ( username = T.text) ?

**Comment by kcchan3012** [comment]
Ya, just now i also google the same thing like what wei chun say.

**Comment by boonsenglim** [comment]
Oh i see, so maybe i try it later and tomorrow let you know if still got any problem.

**Comment by boonsenglim** [comment]
By the way what should I do if the user successfully register and store it to the database?

**Comment by pqyyieeuyi** [comment]
Maybe straight let user log in the account?

**Comment by kcchan3012** [comment]
I dont agree with you ben if the user can straight log in to the account then why we still need the log in function?

**Comment by boonsenglim** [comment]
No, i think what ben mean is let user log in the account when user successful log in to the account so that the user no need press log in function again.

**Comment by Lai0723** [comment]
Ya , i agree with boon seng this will give convenient to user and this only happen when user create new account only, if user log out he or she must log in with the log in function again.

**Comment by pqyyieeuyi** [comment]
Ya, this is what i mean.

**Comment by kcchan3012** [comment]
oh, this seem a good idea.

**Comment by kcchan3012** [comment]
I have no problem anymore.

**Comment by pqyyieeuyi** [comment]
Me too. Boon Seng if tomorrow finished please send to me.

**Comment by boonsenglim** [comment]
Ok, I do it right now.

**Comment by Lai0723** [comment]
So i think that all for today, everyone can dismiss now.
